pages/workspace/tracking-plan/implementation-status.mdx

```diff
@@ -100,5 +100,7 @@ Based on these checks we report the implementation status using one of the follo
 - Event is sometimes sent correctly
 - Event is never sent correctly
 
+> Inspector monitors events from all branches, so it may report 'Sometimes sent correctly' for the correctly implemented events if old version is still sent from main or other branches.
+> In order to make sure the inspector is reporting correctly for your branch, we suggest to decrease the time window to have it catch only a time where you are triggering the events changed.
```

Comment:
Should we keep this out until we have the time window configuration?

Reply:
maybe, can i comment it out somehow?